src/transsystem.c

```diff
@@ -45,18 +45,22 @@ callback_tree_before(T_PTR_tree entry) {
 
   info = (char*) tree_getinfo(entry);
   if (strcmp(info,"rules") == 0) {
+  	printf("Entering rules section\n");
     tree_brk_branch(entry);
     /* nbr_rules will be fill at this point */
-    rulescode_produce(entry, &_system);
+    rulescode_produce(entry, _system);
   } else if (strcmp(info,"init") == 0) {
+  	printf("Entering init section\n");
```

Comment:
I'm going to change it. I have noticed that this pull request is based on an 'old' version of mist without the debug prints.

It's not important because all will work when merging this with the master branch but this pull request can't be compiled itself because of these PRINT